doc/dev/cpu-profiler.rst

```diff
@@ -19,25 +19,17 @@ executing the following::
 Compiling Ceph for Profiling
 ============================
 
-To compile Ceph for profiling, first clean everything. :: 
+To compile Ceph for profiling, first clean everything. ::
 
-	make distclean
+    git clean -dfx
 	
-Then, export the following settings so that you can see callgraph output. :: 
+Finally, compile Ceph. ::
 
-	export CFLAGS="-fno-omit-frame-pointer -O2 -g"
-
-Finally, compile Ceph. :: 
-
-	./autogen.sh
-	./configure
-	make
-
-You can use ``make -j`` to execute multiple jobs depending upon your system. For
-example::
-
-	make -j4
+	./do-cmake.sh -DCMAKE_CXX_FLAGS="-fno-omit-frame-pointer -O2 -g"
+    cd build
+    cmake --build .
 
+In which, the ``CMAKE_CXX_FLAGS`` is specified, so that you can see callgraph output.
```

Comment:
```suggestion
In which, the ``CMAKE_CXX_FLAGS`` is specified, so that you can see callgraph output.
```In this command, ``CMAKE_CXX_FLAGS`` is specified. This causes the command to provide callgraph output.

Reply:
This is the one area where I can unambiguously make a suggestion for improvement. Otherwise, as far as I can tell, this is ready for merge.

Reply:
@zdover23 thank you for the suggestion. but the callgraph output is not generated by "the command" in this context. the callgraph is generated by the oprofile tools. see https://oprofile.sourceforge.io/doc/opreport.html . could you help with a more accurate explanation here?

Reply:
The text I suggested doesn't say that the callgraph output is generated by the command. It says that the command just provides the output.

But let's try to make it more explicit.

We could take "cause" out entirely, and have the sentence say something like:

> In this command, ``CMAKE_CXX_FLAGS`` is specified. This provides callgraph output.

Reply:
@zdover23 thank you! updated and repushed.

Reply:
Qapla!